patches/minecraft/net/minecraft/client/renderer/tileentity/PistonTileEntityRenderer.java.patch

```diff
@@ -1,11 +1,12 @@
 --- a/net/minecraft/client/renderer/tileentity/PistonTileEntityRenderer.java
 +++ b/net/minecraft/client/renderer/tileentity/PistonTileEntityRenderer.java
-@@ -61,8 +61,12 @@
+@@ -61,8 +61,13 @@
     }
  
     private void func_228876_a_(BlockPos p_228876_1_, BlockState p_228876_2_, MatrixStack p_228876_3_, IRenderTypeBuffer p_228876_4_, World p_228876_5_, boolean p_228876_6_, int p_228876_7_) {
 -      RenderType rendertype = RenderTypeLookup.func_239221_b_(p_228876_2_);
-+      net.minecraft.client.renderer.RenderType.func_228661_n_().stream().filter(t -> RenderTypeLookup.canRenderInLayer(p_228876_2_, t)).forEach(rendertype -> {
++      RenderType.func_228661_n_().stream().filter(t -> RenderTypeLookup.canRenderInLayer(p_228876_2_, t)).forEach(rendertype -> {
++      rendertype = RenderTypeLookup.canRenderInLayer(p_228876_2_, RenderType.func_228645_f_()) ? RenderType.func_239269_g_() : rendertype;
```

Comment:
This does not seem correct. This replaces every render type with `TranslucentMovingBlock` if the block can render in the Translucent render type, but I think it should only replace the `Translucent` render layer with `TranslucentMovingBlock`, or am I wrong?

Reply:
Indeed, I'll add a check that this is already in translucent.